### ISSUE TITLE
[1.18.1] Radium together with the Blue Skies mod makes their dimensions not render when entered.

### ISSUE THREAD

**Issue by Gbergz** [Asek3/Radium#1]
Title ^

After entering a dimension from Blue Skies, you will end up in the void and the world won't render / generate around you at all.

### Versions:
**Minecraft:** 1.18.1
**Forge:** 39.0.44
**Radium:** 0.7.7b | https://www.curseforge.com/minecraft/mc-mods/radium-reforged/files/3615873
**Blue Skies:** 1.3.3 | https://www.curseforge.com/minecraft/mc-mods/blue-skies/files/3624825

### Screenshot: 
![image](https://user-images.githubusercontent.com/26634554/151403710-a38b887b-2d50-4b94-ae71-4fea68c04ba6.png)

**-** Gbergz

**Comment by Asek3** [comment]
So, is this problem really caused by Radium?
![image](https://user-images.githubusercontent.com/64613894/151415766-99dc957f-ca9b-4e8f-8dcd-177a4e8e2a4a.png)

**Comment by Gbergz** [comment]
Seems so, can't get it to work in a clean instance either.

**Comment by Asek3** [comment]
> Seems so, can't get it to work in a clean instance either. The issues goes away when this mod is removed from the instance.

Update Forge

**Comment by Gbergz** [comment]
Ah if it is fixed by a newer forge version I can't test that right now. But good to know!

**Comment by Matthysse** [comment]
I'm on the latest forge version and this is still an issue. Maybe caused by the combination with the Starlight mod?

**Comment by Gbergz** [comment]
I didn't have Starlight, so prob not.

**Comment by Matthysse** [comment]
Yeah, tested with only Radium and BlueSkies, and the issue is there. 
![2022-02-08_12 52 18](https://user-images.githubusercontent.com/32510957/152982297-a9cd473a-bd71-4df2-ae3b-06d19bad53e5.png)

**Comment by brucethemoose** [comment]
This is still an issue in 1.19.2:

https://mclo.gs/3e4LODb

Same behavior, I stepped through a portal in a fresh world with just Radium/Blueskies, and the new dimension doesn't render.

Running latest forge and radium builds.

**Comment by brucethemoose** [comment]
Found a fix for this, add `mixin.ai.poi=false` to the config

Just looking at the log, my guess is its probably `'ai.poi.fast_portals.PortalForcerMixin'`?

**Comment by Sebastrn** [comment]
Would this config be in the server or in the client? or both?

**Comment by brucethemoose** [comment]
> Would this config be in the server or in the client? or both?

I'm guessing server, but probably both to be safe?
